lifting_veil/baseline.py

```diff
@@ -38,7 +36,7 @@ def create_agent(sess, environment, summary_writer=None):
     gin.clear_config()
     gin.parse_config_files_and_bindings([gin_file], gin_bindings, skip_unknown=False)
 
-    agent_runner = run_experiment.TrainRunner(LOG_PATH, create_agent, gym_lib.create_gym_environment)
```

Comment:
Don't we need to use the same pattern as [here](https://github.com/JohanSamir/rainbow_extend/blob/main/lifting_veil/main.py#L60) to use minatar and gym?

Reply:
that can also work but is unnecessary, since this parameter is being set in the config files.

Reply:
e.g. https://github.com/JohanSamir/rainbow_extend/blob/main/lifting_veil/Configs/dqn_asterix.gin#L51

Reply:
Oh, good point, that was only needed for offline